src/spartan_tcp_handler.erl

```diff
@@ -39,7 +39,6 @@ callback_mode() ->
     state_functions.
 
 init([Ref, Socket, Transport, Opts]) ->
-    {ok, _} = timer:kill_after(?TIMEOUT * 5),
```

Comment:
I reviewed them. As such they look fine. We might need to watch out if we hit the issue of Spartan getting stuck again